modules/installation-guide/examples/system-variables.adoc

```diff
@@ -155,10 +153,10 @@ pass:[<!-- vale off -->]
 ,=== 
  Environment Variable Name,Default value, Description 
  
- `+CHE_INFRA_OPENSHIFT_PROJECT+`,"","DEPRECATED - please do not change the value of this property otherwise the existing workspaces will loose data. Do not set it on new installations. Defines OpenShift namespace in which all workspaces will be created. If not set, every workspace will be created in a new project, where project name = workspace id It's possible to use <username> and <userid> placeholders (e.g.: che-workspace-<username>). In that case, new project will be created for each user. OpenShift oauth or service account with permission to create new projects must be used. If the project pointed to by this property exists, it will be used for all workspaces. If it does not exist, the namespace specified by the che.infra.kubernetes.namespace.default will be created and used." 
- `+CHE_INFRA_OPENSHIFT_TRUSTED__CA_DEST__CONFIGMAP__LABELS+`,"`+config.openshift.io/inject-trusted-cabundle=true+`","Comma separated list of labels to add to the CA certificates config map in user workspace. See che.infra.kubernetes.trusted_ca.dest_configmap property. This default value is used for automatic cluster CA bundle injection in Openshift 4." 
+ `+CHE_INFRA_OPENSHIFT_PROJECT+`,"","DEPRECATED - Do not change the value of this property otherwise the existing workspaces will loose data. Do not set it on new installations. Defines OpenShift namespace in which all workspaces will be created. If not set, every workspace will be created in a new project, where project name = workspace id It's possible to use `<username>` and `<userid>` placeholders (for example: `che-workspace-<username>`). In that case, new project will be created for each user. OpenShift OAuth or service account with permission to create new projects must be used. If the project pointed to by this property exists, it will be used for all workspaces. If it does not exist, the namespace specified by the `che.infra.kubernetes.namespace.default` will be created and used. This parameter is subject to removal in future releases. Legacy workspaces located in this namespace may become unreachable in future releases. See the documentation for possible next steps." 
```

Comment:
can we link to the doc referenced here? or to a GH issue / JIRA about deprecation / removal?

Reply:
also "loose data" -> should be "lose data". 

Loose ... opposite of tight
Lose ... verb meaning to misplace